### ISSUE TITLE
how to customize my own response serializer?

### ISSUE THREAD

**Issue by leedalei** [cdfmlr/crud#54]
I‘m new in writing golang. It's there a way to use my own response serializer?

**Comment by cdfmlr** [comment]
Yes. You can customize a controller to make it. [sshman](https://github.com/cdfmlr/sshman) shows examples to do so (you can find the related function `UserGetSelf` & `UserGetSessions` in [controller.go](https://github.com/cdfmlr/sshman/blob/bb5f93c4efafe8ed37d60671513b140e21ec1e27/controller.go#L23-L97C2), and they are registered in [router.go](https://github.com/cdfmlr/sshman/blob/bb5f93c4efafe8ed37d60671513b140e21ec1e27/router.go#L55-L56)).
